CONTRIBUTING.md

```diff
@@ -18,6 +18,13 @@ Troubleshooting `pg_config: command not found`: The tests in this package depend
 
 Install the [`aws-cli`](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to publish AWS layers from local.
 
+### Upgrade npm for Node.js v14
+
+If you are using Node.js v14 and would like to take advantage of the "overrides" feature in package.json, you need to upgrade npm manually since Node.js v14 ships with npm v6, and `overrides` were introduced in npm v8.3. Upgrade npm manually to the latest version using the following command:
+```
+npm install npm@latest
```

Comment:
Q: Does this work? I thought we need to use `npm install npm@8.x.x` or `nvm install-latest-npm`?

Reply:
@aryamohanan

Reply:
Perfect. Thanks for testing